### PR TITLE
Modifications of measures #454

### PR DESCRIPTION
**Pull request by arnaud-m**
Just trying a small pull request ;)

### CODE REVIEW

File: src/main/java/org/chocosolver/solver/search/measure/MeasuresRecorder.java [hunk 1]
```diff
@@ -29,413 +29,177 @@
...
-
-    /**
-     * Search state
-     */
-    protected SearchState state;
-
-    //***********************************************************************************
-    // CONSTRUCTOR
-    //***********************************************************************************
+    private long startingTime;
```
Comment (arnaud-m) [inline src/main/java/org/chocosolver/solver/search/measure/MeasuresRecorder.java]: No, it is a private field used in a lambda function.

**Comment by chocotrust** [comment]
I'll add it manually. I may be keep StopWatch name instead of Timer, i'll see

**Comment by chocotrust** [comment]
It seems that you have an indentation problem. 
I will soon provide the file format we use.

**Comment by arnaud-m** [comment]
I tried to modify the "Java Convention" built-in formatter of Eclipse to match yours.
I hope that export will work (see [here](http://stackoverflow.com/questions/36432438/importing-code-style-formatting-settings-into-eclipse-from-intellij-idea)).

**Pull request merged**